### ISSUE TITLE
Don't remove <script></script> in a discussion

### ISSUE THREAD

**Issue by AlxxxlA** [CaliOpen/Caliopen#608]
Hi,

I do a simple test writting a <script>alert('hey');</script> from a caliopen discussion, when I sent the message, it removes the '<script></script>'. I think this is to avoid XSS attack, but the content should be displayed and not removed. If I sent a javascript example to a contact, i don't want that the browser execute the javascript but i want that my contact see the javascript code.

**Comment by sapiens-sapide** [comment]
Yes, we definitively need to improve email sanitation. For now, we've choosen a conservative approach to avoid XSS attack. We'll make the sanitation better.

**Comment by quazardous** [comment]
And maybe add some cool features like markdown formatting ;p
